### PR TITLE
feat:m:major - intergration of mongodg, mongoose and dotenv

### PR DESCRIPTION
**Pull request by Xemko**
! make sure to add .env file to server directory.
add these variables to the .env file ===>

DB_URI=mongodb+srv://NAME:PASSWORD@cluster0.dqiw4gm.mongodb.net/WePlants?retryWrites=true&w=majority

PORT=3000

### CODE REVIEW

**Comment by Xemko** [comment]
run server side app - correct result should be 
image

File: apps/server/src/controllers/main.controller.js [hunk 1]
```diff
@@ -1,8 +1,21 @@
 const mainService = require('../services/main.service');
 
 const signIn = async (req, res) => {
-    // const plants = await mainService.signIn();
-    res.send('sign-in page is under construction');
+    const { phoneNumber } = req.body;
+    if (!phoneNumber) {
+       return res.status(400).send('phoneNumber is required');
```
Comment (tgcaspa) [inline apps/server/src/controllers/main.controller.js]: ok. just return a json
`{ "code": 400, "message": "phoneNumber is required" }`

File: apps/server/src/controllers/main.controller.js [hunk 2]
```diff
@@ -1,8 +1,21 @@
 const mainService = require('../services/main.service');
 
 const signIn = async (req, res) => {
-    // const plants = await mainService.signIn();
-    res.send('sign-in page is under construction');
+    const { phoneNumber } = req.body;
+    if (!phoneNumber) {
```
Comment (tgcaspa) [inline apps/server/src/controllers/main.controller.js]: S: validate the given value. you can copy from client sign-in.validator.ts file

Comment (Xemko) [reply apps/server/src/controllers/main.controller.js]: In progress

Comment (Xemko) [reply apps/server/src/controllers/main.controller.js]: Validation still in progress.

**Comment by Xemko** [comment]
+ basic authentication logic with JWT token, still in progress.
+ json object in return, check and approve .
! Validation is in work. Perhaps JOI Schema for Validation? https://www.npmjs.com/package/joi

**Review by tgcaspa — APPROVED** [review]
remove all console.logs

File: apps/server/src/controllers/main.controller.js (hunk 1 above)
Comment (tgcaspa) [reply apps/server/src/controllers/main.controller.js]: please be align to the MVP requirements and messages...

**Comment by Xemko** [comment]
console.logs for my testing. i'll remove it later

**Pull request merged**